apollo-api/src/main/java/com/apollographql/apollo/api/ResponseFieldMapper.java

```diff
@@ -2,7 +2,9 @@
 
 import java.io.IOException;
 
-/** TODO */
+/**
+ * * ResponseFieldMapper is responsible for mapping the responses returned by the server back to data.
```

Comment:
one *  is redundant.

ResponseFieldMapper is an abstraction for mapping the response data returned by the server back to generated models.